tests/unit/compose/docker-compose-internal-auth_test.go

```diff
@@ -263,7 +263,7 @@ func (s *commonServicesInternalAuthDockerComposeTest) TestServiceEnvironment() {
 				"FEATURE_FLAG_ENABLE_INVITATIONS=false",
 				"FIFTYONE_API_URI=https://example-api.fiftyone.ai",
 				"FIFTYONE_APP_ALLOW_MEDIA_EXPORT=true",
-				"FIFTYONE_APP_TEAMS_SDK_RECOMMENDED_VERSION0.16.0",
+				"FIFTYONE_APP_TEAMS_SDK_RECOMMENDED_VERSION=0.16.0",
```

Comment:
We chomped an equals sign?  Bad alligator.

Reply:
I mean, good eyes, but it was a perl regex, so do you think you caught them all? 😆

Reply:
Not only did I double check by searching, but the failing tests where a clue.